docs/source/tutorials_source/platform/tutorial_sunflowers.py

```diff
@@ -5,8 +5,8 @@
 Tutorial 2: Diversify the Sunflowers Dataset
 =============================================
 
-This tutorial highlights the basic functionality of sampling in the web-app.
-You can use the Coreset sampling algorithm to choose a diverse subset of your dataset.
+This tutorial highlights the basic functionality of selecting in the web-app.
```

Comment:
selecting what? I don't think we can use selecting just as a plain replacement of sampling. It sounds very unnatural. Selecting a subset, selecting images, etc are alternatives.

Reply:
Here too, you didn't make any changes. Did you change things for my other comments?